Baby MonitorTests/Networking/WebRTC/WebRtcServerManagerTests.swift

```diff
@@ -80,4 +95,158 @@ class WebRtcServerManagerTests: XCTestCase {
         // Then
         XCTAssertEqual([streamId], observer.events.map { ($0.value.element as! String) })
     }
+
+    func testShouldNotEmitStreamWhenNoCapturer() {
+        // Given
+        let bag = DisposeBag()
+        let scheduler = TestScheduler(initialClock: 0)
+        let observer = scheduler.createObserver(MediaStream.self)
+        let peerConnection = PeerConnectionMock()
+        let streamId = "test"
+        let peerConnectionFactory = PeerConnectionFactoryMock(peerConnectionProtocol: peerConnection,
+                                                              videoCapturer: nil,
+                                                              mediaStream: streamId as MediaStream)
+        let connectionDelegateProxy = PeerConnectionProxyMock()
+        let sut = WebRtcServerManager(peerConnectionFactory: peerConnectionFactory,
+                                      connectionDelegateProxy: connectionDelegateProxy,
+                                      scheduler: AsyncSchedulerMock())
+
+        sut.mediaStream
+            .subscribe(observer)
+            .disposed(by: bag)
+
+        // When
+        sut.start()
+
+        // Then
+        XCTAssertEqual([], observer.events.map { ($0.value.element as! String) })
```

Comment:
Maybe flip it around for better visibility? : `XCTAssertEqual(observer.events.map { ($0.value.element as! String) }, [])`

Reply:
I agree that visibility would be much better, but still I think it's nice to keep the `expectedResult` like a first parameter as usually it's done :)